### ISSUE TITLE
文本遗漏

### ISSUE THREAD

**Issue by DandelionI** [dotnet/AspNetCore.Docs.zh-cn#348]
[在此处输入反馈]
**Student 实体** 一项内，_有关详细信息，请参阅 F Core - 密钥。_ 中应为 _EF Core_

---
#### 文档详细信息

⚠ *请勿编辑此部分。 docs.microsoft.com ➟ GitHub 问题链接必须具有此部分。*

* ID: 7a54c97f-ddd4-b041-5e5f-a2df009ace50
* Version Independent ID: fa488196-1f98-08ea-adcc-17d9dac86727
* Content: [ASP.NET Core 中的 Razor Pages 和 Entity Framework Core - 第 1 个教程（共 8 个）](https://docs.microsoft.com/zh-cn/aspnet/core/data/ef-rp/intro?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/data/ef-rp/intro.md](https://github.com/dotnet/AspNetCore.Docs.zh-cn/blob/live/aspnetcore/data/ef-rp/intro.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
[Enter feedback here]
**Student entity** . For more information, see F Core-Key. Should be EF Core

@guardrex @scottaddie  @ajcvickers  should I add `EF Core` to the no-loc metadata? Is that a problem for your docs?

**Comment by guardrex** [comment]
That is one that I **_don't_** code fence.

**Comment by guardrex** [comment]
I have 25 of them ("EF Core") across three topics.

**Comment by guardrex** [comment]
I think the whole repo has several hundred of them ... perhaps over 300.

**Comment by Rick-Anderson** [comment]
> I think the whole repo has several hundred of them ... perhaps over 300.

So adding it to the no-loc should fix it, right?

**Comment by guardrex** [comment]
Yes ... it needs quotes for the entry given the presence of the space there.
